modules/using-the-interactive-installer.adoc

```diff
@@ -9,6 +9,8 @@
 [role="_abstract"]
 Use the interactive installer to generate the required secrets, deployment configurations, and deployment scripts for your environment.
 
+#Do we need to put a note here that enabling Scanner V4 isn't supported with the roxctl installation? (Is that true?)#
```

Comment:
It is supported per meeting on 2/28 with scanner team. will adjust docs accordingly.